Makefile

```
@@ -13,6 +13,9 @@ version=v${build_date}-${git_commit}
 SOURCE_GIT_TAG=v1.0.0+$(shell git rev-parse --short=7 HEAD)
 
 GO_LD_EXTRAFLAGS=-X github.com/openshift-eng/jira-lifecycle-plugin/vendor/k8s.io/client-go/pkg/version.gitCommit=$(shell git rev-parse HEAD) -X github.com/openshift-eng/jira-lifecycle-plugin/vendor/k8s.io/client-go/pkg/version.gitVersion=${SOURCE_GIT_TAG} -X sigs.k8s.io/prow/pkg/version.Name=jira-lifecycle-plugin -X sigs.k8s.io/prow/pkg/version.Version=${version}
+GOLINT=golangci-lint run
+
+lint: verify-golint
```

checkmake (0.3.2)

[warning] Target "lint" should be declared PHONY.

(phonydeclared)